### DISCUSSION TITLE
Set Custom ObjectMapper

### DISCUSSION THREAD

**Discussion by aishwary32402** [networknt/json-schema-validator#1117] [Q&A]
Hi All, Is the support to use the custom ObjectMapper has been removed? I can see the methods of jsonMapper and yamlMapper as deprecated.
I want to set the ObjectMapper to use BIGDECIMAL for floats and Long for Ints. Is there any way I can achieve that?

**Comment by justin-tay** [comment]
Configure a `JsonNodeReader` with the `ObjectMapper` you want.
